pom.xml

```diff
@@ -79,6 +79,8 @@
         <maven.jar.version>3.1.0</maven.jar.version>
         <maven.javadoc.version>3.0.1</maven.javadoc.version>
         <jacoco.plugin.version>0.8.2</jacoco.plugin.version>
+        <junit.jupiter.version>5.1.0</junit.jupiter.version>
```

Comment:
why not the lastest version?
https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-engine/5.3.2

Reply:
That's right, @otaviojava , I probably overlooked this while pasting the dependencies from somewhere else. Updated both dependencies to their latest version.